tests/PatchEventTest.php

```diff
@@ -11,7 +11,7 @@
 use cweagans\Composer\PatchEvents;
 use Composer\Package\PackageInterface;
 
-class PatchEventTest extends \PHPUnit_Framework_TestCase {
```

Comment:
this change was required to get the test running on a more modern phpunit version